kairon/async_callback/utils.py

```
@@ -0,0 +1,404 @@
+            decrypted_data = jsond.loads(decrypted_bytes.decode("utf-8"))
+
+            response_dict = {
+                "decryptedBody": decrypted_data,
+                "aesKeyBuffer": aes_key,
+                "initialVectorBuffer": iv,
+            }
+
+            return response_dict
+
+        except Exception as e:
+            raise Exception(f"decryption failed-{str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)